Godeps/_workspace/src/k8s.io/kubernetes/cmd/kubelet/app/server.go

```diff
@@ -148,6 +148,9 @@ type KubeletServer struct {
 
 	KubeApiQps   float32
 	KubeApiBurst int
+
+	// Pull images one at a time.
+	SerializeImagePulls bool
```

Comment:
Needs to go into the release notes, and a description of how to turn it off (example of config change)